src/NzbDrone.Common/Disk/IDiskProvider.cs

```diff
@@ -25,13 +25,9 @@ public interface IDiskProvider
         long GetFolderSize(string path);
         long GetFileSize(string path);
         void CreateFolder(string path);
-        void CopyFolder(string source, string destination);
-        void MoveFolder(string source, string destination);
-        void TransferFolder(string source, string destination, TransferMode transferMode);
         void DeleteFile(string path);
-        void CopyFile(string source, string destination, bool overwrite = false);
-        void MoveFile(string source, string destination, bool overwrite = false);
-        TransferMode TransferFile(string source, string destination, TransferMode transferMode, bool overwrite = false);
+        void CopySingleFile(string source, string destination, bool overwrite = false);
+        void MoveSingleFile(string source, string destination, bool overwrite = false);
```

Comment:
At first I just needed a separate name to refactor easily. Just kept it coz it would be a gentle push for other devs to use the TransferService. But I'm okay either way.